test/coverband/collectors/delta_test.rb

```diff
@@ -48,4 +48,30 @@ def setup
     results = Coverband::Collectors::Delta.results(mock_coverage(current_coverage))
     assert_equal(current_coverage, results)
   end
+
+  test 'one shot lines results' do
+    unless Coverband.configuration.one_shot_coverage_implemented_in_ruby_version?
+      module ::Coverage
```

Comment:
Instead of monkey patching here, maybe just not run this test in ruby < 2.6 ?

Reply:
It's not clear to me where the setup is for what versions of ruby are tested.  As long as it is actually tested on versions of ruby >= 2.6, that seems fine.

What did you have in mind though?
something like
```
if Coverband.configuration.one_shot_coverage_implemented_in_ruby_version?
  test 'one shot lines results' do
    ...
  end
end
```

Reply:
Yes, I think that is simpler.

Reply:
Done, the test is now run conditionally.